Library/Homebrew/os/linux.rb

```diff
@@ -19,6 +19,34 @@ def os_version
         "Unknown"
       end
     end
+
+    def host_gcc_version
+      gcc = Pathname.new "/usr/bin/gcc"
+      return unless gcc.executable?
+
+      `#{gcc} --version 2>/dev/null`[/ (\d+\.\d+\.\d+)/, 1]
+    end
+
+    def kernel_headers_version
```

Comment:
Let's 🔥 this for now as there hasn't been any pressing needs for this before this PR and it's all new code.